templates/ep19/bs/profiles/profile_unavailable.html

```diff
@@ -0,0 +1,18 @@
+{% extends "ep19/bs/base.html" %}
+
+{% block content %}
+<div class="container page" id="speaker_page">
+    <nav aria-label="breadcrumb">
+        <ol class="breadcrumb">
+            <li class='breadcrumb-item'><a href="/">Home</a></li>
+            <li class='breadcrumb-item'><a href='#'>Talks</a></li>
```

Comment:
looks like a leftover. :)

Reply:
This was actually left on purpose because I wasn't sure where to put this page :sweat_smile: what do you suggest?